src/Entity/MetadataDisplayEntity.php

```diff
@@ -138,6 +142,10 @@ class MetadataDisplayEntity extends ContentEntityBase implements MetadataDisplay
 
   // Implements methods defined by EntityChangedInterface.
   use EntityChangedTrait;
+  use UseCacheBackendTrait;
+  use LoggerChannelTrait;
+
+  CONST CACHE_TAG_ID = 'format_strawberry:metadata_display_related_tags:';
```

Comment:
@DiegoPino Should this be:

```suggestion
  CONST CACHE_TAG_ID = 'format_strawberryfield:metadata_display_related_tags:';
```

for consistency? Or was it deliberately named that way? Does it make any difference?

Reply:
@aksm I kept the prefix shorter bc the view mode resolver and the embargo are using already that prefix. And even if your suggestion is better naming, I would have to change all the other tags and generate a hook update to recreate them under a new Cache bin on update. Feels right now a bit too much.

Reply:
Ok, that makes sense. Thanks.